doc/user/posix.md

```diff
@@ -43,62 +43,62 @@ vice versa.
 The dfuse daemon runs a single instance per node to provide a user POSIX access
 to DAOS, and it should be run with the credentials of the user and typically will
 be started and stopped on each compute node as part of the prolog and epilog
-scripts of any resource manager or scheduler in use.  One dfuse daemon per node
+scripts of any resource manager or scheduler in use.  One DFuse daemon per node
 can process requests for multiple clients.
 
-A single dfuse instance can provide access to multiple pools and containers
+A single DFuse instance can provide access to multiple pools and containers
 concurrently, or can be limited to a single pool, or a single container.
 
 ### Restrictions
 
 DFuse is limited to a single user. Access to the filesystem from other users,
 including root will not be honored, and as a consequence of this, the chown
-and chgrp calls are not supported.  Hard links and special device files, except 
+and chgrp calls are not supported.  Hard links and special device files, except
 symbolic links, are not supported, nor are any ACLs.
 
 DFuse can run in the foreground, keeping the terminal window open, or it can
 daemonize to run like a system daemon, however, to do this and still be
 able to access DAOS it needs to daemonize before calling daos_init() which in
 turns means it cannot report some kinds of startup errors either on
-stdout/stderr or via its return code.  When initially starting with dfuse it
-is recommended to run in foreground mode (--foreground) to better observe
+stdout/stderr or via its return code.  When initially starting with DFuse it
+is recommended to run in foreground mode (`--foreground`) to better observe
 any failures.
 
-Inodes are managed on the local node by the dfuse, so while inode numbers
+Inodes are managed on the local node by the DFuse, so while inode numbers
 will be consistent on a node for the duration of the session, they are not
-guaranteed to be consistent across restarts of dfuse or across nodes.
+guaranteed to be consistent across restarts of DFuse or across nodes.
 
-It is not possible to see pool/container listings through dfuse, so if readdir,
-ls or others are used for this, dfuse will return ENOTSUP.
+It is not possible to see pool/container listings through DFuse, so if readdir,
+ls or others are used for this, DFuse will return ENOTSUP.
 
 ### Launching
 
-dfuse should be run with the credentials (user/group) of the user that will
+DFuse should be run with the credentials (user/group) of the user that will
 be accessing it, and that owns any pools that will be used.
 
 There are two mandatory command-line options, these are:
 
-* --svc=RANKS  <service replicas>
-* --mountpoint=PATH <path to mount DAOS>
+* --svc=RANKS  <service replicas\>
```

Comment:
Maybe these command line option lists should be formatted as tables, instead of having the description in pointy brackets.